### PR TITLE
Uniform use of `pytz` timezones.

### PR DESCRIPTION
**Pull request by Froskekongen**
This PR makes sure that all instances of the `UTC` timezone are `pytz` timezones. This fixes https://github.com/quantopian/trading_calendars/issues/45

### CODE REVIEW

File: trading_calendars/trading_calendar.py [hunk 1]
```diff
@@ -41,9 +42,9 @@
 from .utils.preprocess import preprocess, coerce
 
 
-start_default = pd.Timestamp('1990-01-01', tz='UTC')
-end_base = pd.Timestamp('today', tz='UTC')
-# Give an aggressive buffer for logic that needs to use the next trading
+start_default = pd.Timestamp('1990-01-01', tz=UTC)
+end_base = pd.Timestamp('today', tz=UTC)
+# Give an aggressive b tuffer for logic that needs to use the next trading
```
Comment (richafrank) [inline trading_calendars/trading_calendar.py]: typo

File: .travis.yml [hunk 2]
```diff
@@ -4,12 +4,12 @@ sudo: false
 
 python:
   - 2.7
```
Comment (richafrank) [inline .travis.yml]: We still want to tests 2.7 with NUMPY_VERSION=1.11.3 PANDAS_VERSION=0.18.1. (Taken from [zipline](https://github.com/quantopian/zipline/blob/013cd1b0ef916024a7233301a2c127111e889434/.travis.yml#L47))

File: .travis.yml [hunk 3]
```diff
@@ -4,12 +4,12 @@ sudo: false
...
-  - 3.5
```
Comment (richafrank) [inline .travis.yml]: We still want to test 3.5 with NUMPY_VERSION=1.11.3 PANDAS_VERSION=0.18.1.

File: .travis.yml (hunk 2 above)
Comment (Froskekongen) [reply .travis.yml]: The travis CI build works, but the appveyor build is bugged. I can change the travis build, but the appveyor build won't work until the underlying bug gets fixed.

Also, I would suggest updating zipline to require newer versions of both pandas and numpy.

Comment (richafrank) [reply .travis.yml]: > the appveyor build is bugged

Oh, is it a particular job in the matrix that's broken?

> Also, I would suggest updating zipline to require newer versions of both pandas and numpy.

Agreed - PRs welcome 😄 .

Comment (Froskekongen) [reply .travis.yml]: In appveyor, the pandas version is not constrained correctly from the specification, so it tries using pandas 0.24.1 with numpy 1.11.3, which are incompatible.

**Comment by Froskekongen** [comment]
@richafrank: There was no explicit need for testing with pandas 0.19. The problem is in the appveyor build, which I outlined in the other conversation.

File: .travis.yml (hunk 2 above)
Comment (richafrank) [reply .travis.yml]: Ah, looks like the `conda create -n testenv` step is missing `pandas=%PANDAS_VERSION%`

Comment (Froskekongen) [reply .travis.yml]: Thanks, and fixed (:

**Comment by richafrank** [comment]
Thanks @Froskekongen !

**Pull request merged**